docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
-chainer
-scikit-image
-scikit-learn
+chainer=1.24
+Cython
 matplotlib
+numpy
+pillow
```

Comment:
Both `Cython` and `Pillow` start with an upper character in pypi. Why are you using upper character only for `Cython`?